portfolio/src/main/webapp/script.js

```diff
@@ -43,4 +43,14 @@ function addRandomQuote() {
     });
 }
 
+/** Adds an HTML element from Servlet */
+function addElement() {
```

Comment:
The function name should be indicative of its purpose. It isn't really adding any elements, just setting the text of an element. Also, this is fetching some data from the backend so it should really be more about that. `fetchGreeting` or `fetchAndDisplayGreeting` is probably better.